src/query-compiler/default-query-compiler.ts

```diff
@@ -1739,3 +1748,13 @@ const JOIN_TYPE_SQL: Readonly<Record<JoinType, string>> = freeze({
   LateralLeftJoin: 'left join lateral',
   Using: 'using',
 })
+
+const OR_CONFLICT_RESOLUTION: Readonly<
+  Record<OrConflictNodeResolutions, string>
+> = freeze({
+  Abort: 'abort',
+  Fail: 'fail',
+  Ignore: 'ignore',
+  Replace: 'replace',
+  Rollback: 'rollback',
+})
```

Comment:
See other comments.

This could've been avoided by just using lowercase everywhere.